activation/activation.go

```diff
@@ -548,7 +551,7 @@ func (b *Builder) BuildNIPostChallenge(ctx context.Context, nodeID types.NodeID)
 	}
 	publish := current + 1
 	metrics.PublishOntimeWindowLatency.Observe(until.Seconds())
-	wait := buildNipostChallengeStartDeadline(b.poetRoundStart(current), b.poetCfg.GracePeriod)
+	wait := b.poetRoundStart(current).Add(-b.poetCfg.GracePeriod)
```

Comment:
Please revert that change, there is a reason we add jitter to `wait` otherwise all nodes on the network will DDOS our PoETs!

Reply:
@fasmat, This jitter was initially created to spread out cals to poet's /submit endpoint. I think it's not useful anymore, at least not in this place, because other things in between here and submitting to poet (i.e positioning ATX lookup, poet certification etc.) should bring enough random jitter. 

Also, poet has been greatly improved since then and I think it should be able to handle it (also, there are retries just in case). If we still want to keep an explicit jitter, I think it should be placed right before submitting to poet.

Wdyt?

Reply:
Alright, convinced 🙂

Reply:
But then also remove the function `buildNipostChallengeStartDeadline`

Reply:
It is used in 4 places, so I will just fix the function, it is better than write same calculations N times